### PR TITLE
CODE RUB: Change Student Guardian Filler method order

### PR DESCRIPTION
**Pull request by rcruzfreelance**
Closes #[626](https://github.com/hassanhabib/OtripleS/issues/626)

### CODE REVIEW

File: OtripleS.Web.Api.Tests.Unit/Services/Foundations/StudentGuardians/StudentGuardianServiceTests.cs [hunk 1]
```diff
@@ -76,11 +58,32 @@ public static IEnumerable<object[]> InvalidMinuteCases()
             };
         }
 
-        private static int GetRandomNumber() => new IntRange(min: 2, max: 10).GetValue();
-        private static int GetNegativeRandomNumber() => -1 * GetRandomNumber();
-        private static string GetRandomMessage() => new MnemonicString().GetValue();
+        private static int GetRandomNumber() =>
+            new IntRange(min: 2, max: 10).GetValue();
+        private static int GetNegativeRandomNumber()
+            => -1 * GetRandomNumber();
```
Comment (hassanhabib) [inline OtripleS.Web.Api.Tests.Unit/Services/Foundations/StudentGuardians/StudentGuardianServiceTests.cs:64]: Move fat arrows up by one

**Comment by hassanhabib** [comment]
@rcruzfreelance you've got some merge conflicts here.

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

**Comment by ShriHumrudha** [comment]
> Closes #[626](https://github.com/hassanhabib/OtripleS/issues/626)

Just for your learning. The branch name has foundations. A tip for the name is similar to the Pull request name and not tied to the layer you are editing.

**Comment by rcruzfreelance** [comment]
Thank you @ShriHumrudha for the tip.

**Comment by rcruzfreelance** [comment]
so it should be like users/rcruzfreelance/code-rub-student-guardian-order?

**Pull request merged**